projects/igniteui-angular/src/lib/radio/themes/dark/_fluent.scss

```diff
@@ -0,0 +1,16 @@
+@use 'igniteui-theming/sass/bem' as *;
+@use 'igniteui-theming/sass/color/functions' as *;
+@use 'styles/themes/standalone' as *;
+@use '../light/tokens' as *;
+
+@include themed-block(igx-radio, fluent, dark) {
+    @include m(invalid) {
+        &:hover {
+            @include e(composite) {
+                &::before {
+                    background: color($color: 'error', $variant: 500);
+                }
+            }
+        }
+    }
```

Comment:
This looks off compared to the WC component implementation:
<img width="945" alt="Screenshot 2025-02-07 at 08 50 36" src="https://github.com/user-attachments/assets/dd748675-bdc7-4f00-83cb-31bcf9e71d15" />